PSKoans/Public/Show-Karma.ps1

```diff
@@ -34,6 +34,11 @@ function Show-Karma {
         [switch]
         $Contemplate,
 
+        [Parameter(Mandatory, ParameterSetName = 'OpenFile')]
+        [Alias('Continue')]
+        [switch]
+        $OpenFile,
```

Comment:
🤔 Not sold on the name here as yet. I'm also thinking we could just roll this into the `-Contemplate` set as an optional switch, so the syntax would end up being `-Contemplate -File` or something?

Reply:
I think "-Contemplate -File" is a little bit too complicated and hard to find for such a useful switch, don't you think?  We can even merge these 2 into just Contemplate, after all there is no need to just open the folder when you can open the file and jump to the line. 😉

Reply:
Hmm. That's a good point, it may be worth doing that.